packages/block-library/src/spacer/edit.js

```diff
@@ -51,50 +61,122 @@ const SpacerEdit = ( {
 		setIsResizing( false );
 	};
 
-	return (
-		<>
-			<View { ...useBlockProps() }>
+	const handleOnHorizontalResizeStop = ( event, direction, elt, delta ) => {
+		onResizeStop();
+		const spacerWidth = Math.min(
+			parseInt( width + delta.width, 10 ),
+			MAX_SPACER_WIDTH
+		);
+		updateWidth( spacerWidth );
+		setIsResizing( false );
+	};
+
+	const resizableBoxWithOrientation = ( blockOrientation ) => {
+		if ( blockOrientation === 'horizontal' ) {
+			return (
 				<ResizableBox
 					className={ classnames(
 						'block-library-spacer__resize-container',
+						'resize-horizontal',
 						{
 							'is-selected': isSelected,
 						}
 					) }
 					size={ {
-						height,
+						width,
+						height: 32,
```

Comment:
Awesome PR, this is going to allow so many new designs! 🎉 

I'm wondering if there might be cases where 32px is too much for a fixed height. Although adjusting vertically to the parent height is trickier than doing so horizontally, ideally, it would behave like the vertical spacing, where width is omitted. Maybe something maybe worth exploring later, right?

Reply:
Solid observation. 24px feels like a plenty tappable size that also goes well with the "minimum" size of the navigation block, as it's the size of the appender.